### DISCUSSION TITLE
How obtain all push from an user

### DISCUSSION THREAD

**Discussion by sjarrossay** [python-gitlab/python-gitlab#2197] [Q&A]
Hello,

According to the documentation to obtain all push made by an user,
I was thinking that I had to call 
`gl.events.list(all=True, action='pushed',sudo='jdoe')`
But  I got nothing, so I think I'm wrong
Can you help me in this way ?

Thanks for all you great job and for your futur answer

**Comment by nejch** [comment]
@sjarrossay are you doing this using an administrator token?

**Reply by sjarrossay** [reply]
Yes, I use an administrator token
